src/app/keyboard/services/tv2-key-binding-matcher.service.ts

```diff
@@ -0,0 +1,86 @@
+import { KeyBindingMatcher } from '../abstractions/key-binding-matcher.service'
+import { KeyBinding } from '../models/key-binding'
+
+export class Tv2KeyBindingMatcher implements KeyBindingMatcher {
+  public isMatching(keyBinding: KeyBinding, keystrokes: string[], isKeyReleased: boolean): boolean {
```

Comment:
`isKeyReleased` is a boolean flag. Would be nice to not have boolean flags.

Reply:
I have changed it to use an enum called `KeyEventType` which has the values `PRESSED` and `RELEASED`.